next.config.js

```diff
@@ -2,4 +2,12 @@ module.exports = {
   webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
     return config;
   },
+  async rewrites() {
+    return [
+      {
+        source: '/api/:slug*',
+        destination: 'https://api.zetk.in/v1/:slug*',
```

Comment:
The URL to the Zetkin API should be externalized eventually, so that we can toggle between using the development environment and the live environment.